service-registry/src/main/java/org/apache/servicecomb/serviceregistry/config/ServiceRegistryConfig.java

```diff
@@ -133,10 +131,8 @@ public boolean isSsl() {
     uriList.forEach(anUriList -> {
       try {
         URI uri = new URI(anUriList);
-        StringBuilder sb = new StringBuilder(uri.getHost());
-        sb.append(':').append(uri.getPort() < 0 ? PROTOCOL_HTTP_PORT : uri.getPort());
         this.ssl = uri.getScheme().startsWith("https");
-        ipPortList.add(NetUtils.parseIpPort(sb.toString()));
+        ipPortList.add(NetUtils.parseIpPort(uri.getScheme(), uri.getAuthority()));
```

Comment:
Can you add a UT for getIpPort and https schema?

Reply:
done